README.md

```diff
@@ -1,33 +1,56 @@
 # WP DETECT
 A WordPress detection tool, detects if a website is running WordPress. wpdetect is a great tool when you just want to check WordPress' presence but do not want to scan the site for vulnerabilities or issues.  
 ### Installation
+
+#### Installing from pip
 You can install wpdetect using pip,
 ```sh
 pip install wpdetect
 ```
 wpdetect requires Python 3 or above to run. If you have Python 2 installed too, make sure to use the right pip.
 
+
 ### Usage
 Syntax
+
+
+```sh
+wpdetect --url https://www.malcare.com/
+```
+
+
 ```sh
-wpdetect <website_url>
+cat URLs.txt | wpdetect
 ```
-Example
+
 ```sh
-wpdetect https://iamlizu.com/
+echo https://hackerone.com | wpdetect
 ```
-Or feed a text file with a list of domains, each domain should be separated with new lines.
+
+
 ```sh
-wpdetect -f sites.txt
+wpdetect --file urls.txt
 ```
-Where `sites.txt` will contain domains like this,
+
+
+Where `urls.txt` will contain domains like this,
 ```sh
-https://iamlizu.com/
+https://www.malcare.com/
 https://www.newyorker.com/
 http://www.techcrunch.com/
 ```
 
+
 Please note that, it is not always possible to detect the presence of WordPress, website admins can take extra measures to remove sign of WordPress.
 
-### What's new in version 1.3.6
-* Fixed minor bugs
\ No newline at end of file
+## Options
```

Comment:
Very nice. I have seen that you included some other options in the help menu in script. Please add those as well?

And instead of modifying how the package is consumed entirely, please add the new options in accord to the old usages ways. 

Adding new options or modes are very welcomed. But user should be able to execute it like `wpdetect wordpress.org` or `wpdetect -f urlList.txt`.

Reply:
Implemented